### DISCUSSION TITLE
ZEN32 Config Browser Missing Parameter 27

### DISCUSSION THREAD

**Discussion by karwosts** [zwave-js/zwave-js#7453] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?

Browsing https://devices.zwave-js.io/?jumpTo=0x027a:0x7000:0xa008:0.0#, I don't see config parameter 27 listed for the Zooz Zen32 scene controller.

This option is mentioned on their advanced options documentation page: https://www.support.getzooz.com/kb/article/608-zen32-scene-controller-advanced-settings/, and there haven't been any new firmware releases lately.

Is this just missing from the configuration file?

> Enable/disable LED blinking when buttons are pressed
> Parameter 27: Choose if you want the LED indicators to flash whenever a button is pressed on the device to confirm scene activation.
> Values: 0 - LED blinking enabled; 1 - LED blinking disabled. Default: 0
> Size: 1 byte dec

What did you expect to happen instead?

Parameter 27 listed at https://devices.zwave-js.io/?jumpTo=0x027a:0x7000:0xa008:0.0#  (and ultimately to be able to control it in home assistant ZwaveJS)

  2. Click on '...'
  3. Scroll down to '...'

Anything else we should know?

### Software versions

Driver (node-zwave-js): ...

Z-Wave JS UI: ...

Home Assistant Z-Wave Integration: ...

Home Assistant Z-Wave JS Addon: ...

ioBroker.zwave2 Adapter: ...

If you are using something non-standard, tell us here: ...

### Z-Wave Controller (Stick/Dongle/...)

discussions

### Device information

Manufacturer: ...
Model name: ...
Node ID: ...

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [ ] The log includes the problematic interaction with the device _(if applicable)_.

- [ ] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

No logfile as its just an issue with the config browser.

[a comment by an automated account is not shown]

**Comment by AlCalzone** [comment, marked as answer]
Will be solved by https://github.com/zwave-js/node-zwave-js/pull/7540
